### PR TITLE
cg-provision is modified to support installation in non gov regions

### PR DESCRIPTION
**Pull request by s-matyukevich**

### CODE REVIEW

**Comment by mogul** [comment]
cc @jmcarp to help resolve conflicts with work already done

**Comment by s-matyukevich** [comment]
I made bucket names unified and resolved conflicts with master branch, @jmcarp  can you please check?

File: ci/pipeline.yml [hunk 1]
```diff
@@ -282,8 +284,8 @@ jobs:
      TF_VAR_concourse_staging_rds_password: {{concourse_staging_rds_password}}
      TF_VAR_concourse_staging_cidr: {{concourse_staging_cidr}}
      TF_VAR_remote_state_bucket: {{aws_s3_tfstate_bucket}}
-     TF_VAR_staging_private_bucket: {{staging_private_bucket}}
-     TF_VAR_prod_private_bucket: {{prod_private_bucket}}
+     TF_VAR_varz_staging_bucket: {{varz_staging_buckett}}
```
Comment (jmcarp) [inline ci/pipeline.yml]: buckett -> bucket

**Comment by jmcarp** [comment]
Thanks for reconciling the bucket names @s-matyukevich. I'm less familiar with the other steps you updated, so it's probably best for @LinuxBozo or @sharms to review and merge.

**Comment by jmcarp** [comment]
Lots of this has happened in other work since this was submitted. Closing.

**Pull request closed without merging**